asciidoc/modules/con_mg_changes-in-vertx-web-graphql.adoc

```diff
@@ -20,3 +20,7 @@ The following methods have been updated and are now supported on polyglot enviro
 * `VertxDataFetcher` is now a factory, use the method `io.vertx.ext.web.handler.graphql.schema.VertxDataFetcher.create()` instead.
 
 * `VertxPropertyDataFetcher` is now a factory, use the method  `io.vertx.ext.web.handler.graphql.schema.VertxPropertyDataFetcher.create()` instead.
+
+== Handling of Post data in GraphQL
+
+In prior releases, Graphql used to handle limited post data. From {VertX} {4} onward, GraphQL handles its own data during redirect and multipart upload operations.XXX
```

Comment:
@vietj Does this note read correct?

Original note:
https://github.com/vert-x3/wiki/wiki/4.0.0-Deprecations-and-breaking-changes#dependency-on-bodyhandler

Reply:
@tsegismont can you address this concern ?

Reply:
@sangeetaraghu this would be better:

_In prior releases, the Vert.x Web GraphQL handler could process `POST` requests without the help of the Vert.x Web `BodyHandler`. The implementation was limited and susceptible to DDoS attacks._
_From {VertX} {4} onward, the `BodyHandler` is required and must be installed before the Vert.x Web GraphQL handler in order to process `POST` requests._

Reply:
@tsegismont Rephrased a bit. Please let me know if it reads ok:
== Handling POST requests in {VertX} Web GraphQL

In prior releases, the {VertX} Web GraphQL handler could process its own POST requests. It did not need {VertX} Web `BodyHandler` to process the requests. However, this implementation was susceptible to DDoS attacks.

From {VertX} {4} onward, to process POST requests `BodyHandler` is required. You must install `BodyHandler` before installing {VertX} Web GraphQL handler.

Reply:
LGTM @sangeetaraghu !

Reply:
Thank you so much @tsegismont